FirebaseDatabase/Sources/Public/FirebaseDatabase/FIRDatabaseQuery.h

```diff
@@ -131,6 +131,10 @@ NS_SWIFT_NAME(DatabaseQuery)
                       withCancelBlock:
                           (nullable void (^)(NSError *error))cancelBlock;
 
+- (void)getDataWithCompletionBlock:
+    (void (^_Nonnull)(NSError *__nullable error,
+                      FIRDataSnapshot *snapshot))block;
```

Comment:
I played around with this as well and it seems like the Swift renaming doesn't work here (maybe @ryanwilson knows why). I was able to get a Swifty name by adding ` NS_SWIFT_NAME(getData(_:))` or `NS_SWIFT_NAME(getData(completion:))` as stated in the API proposal.

Reply:
Yep this will need a manual naming: it doesn't match the expected completion handler style of (`SomeType`, `NSError`) so the automagic conversion won't work.

Reply:
I've added the swift name so that it's consistent with go/rtdb-get.